webapp/src/mutator.ts

```diff
@@ -392,10 +396,33 @@ class Mutator {
 
         const oldBlocks: IBlock[] = [board]
         const newBlocks: IBlock[] = [newBoard]
-        if (propertyTemplate.type === 'select') {
+
+        // We are going from a select to a multiSelect or multiSelect to select
+        if ((propertyTemplate.type === 'select' && type === 'multiSelect') || (propertyTemplate.type === 'multiSelect' && type === 'select')) {
+            for (const card of boardTree.allCards) {
+                const oldValue = Array.isArray(card.properties[propertyTemplate.id]) ?
+                    (card.properties[propertyTemplate.id].length > 0 && card.properties[propertyTemplate.id][0]) :
+                    card.properties[propertyTemplate.id]
+                if (oldValue) {
+                    const newValue = propertyTemplate.options.find((o) => o.id === oldValue)?.id
+                    const newCard = new MutableCard(card)
+
+                    if (newValue) {
+                        newCard.properties[propertyTemplate.id] = type === 'multiSelect' ? [newValue] : newValue
+                    } else {
+                        // This was an invalid select option, so delete it
+                        delete newCard.properties[propertyTemplate.id]
+                    }
+
+                    newBlocks.push(newCard)
+                    oldBlocks.push(card)
+                }
+                newTemplate.options = propertyTemplate.options
+            }
+        } else if (propertyTemplate.type === 'select' || propertyTemplate.type === 'multiSelect') { // If the old type was either select or multiselect
             // Map select to their values
             for (const card of boardTree.allCards) {
-                const oldValue = card.properties[propertyTemplate.id]
+                const oldValue = Array.isArray(card.properties[propertyTemplate.id]) ? card.properties[propertyTemplate.id][0] : card.properties[propertyTemplate.id]
```

Comment:
Do you need to check length here, as done above?

Reply:
Yep! I don't think it hurts to have it. I believe I missed it here. Good catch.

Reply:
This changed helped me realize and removed the entire if block above!